worker/jobTypes/S3Publish.js

```diff
@@ -1,33 +1,30 @@
 const fs = require('fs-extra');
-const workerUtils = require('../utils/utils');
+const workerUtils = require('../utils/utils')
```

Comment:
Did the linter suggest removing the ";"?

Reply:
yeah the linter wanted me to remove all semicolons - but i didnt do it for all of them, just this one. i can add it back and then merge to stage.